config/config_8902.toml

```diff
@@ -33,6 +33,7 @@ compact_trigger_cold_duration = "1h"
 max_compact_size = "2G" # 2147483648
 max_concurrent_compaction = 4
 strict_write = false
+reserver_space = "0"
```

Comment:
Does `reserver` spells right? You may want to use `reserve`.

Reply:
It means the minimum space reserves in RAM or hark disk?

Reply:
you are right , use reserve. It means the minimum size reserves in disk.